src/views/shared/transaction-info/transaction-info.view.jsx

```diff
@@ -74,7 +76,9 @@ function TransactionInfo ({
               value: getPartiallyHiddenHermezAddress(txData.fromHezEthereumAddress)
             }}
             to={{
-              subtitle: getPartiallyHiddenHermezAddress(txData.toHezEthereumAddress)
+              subtitle: txData.toHezEthereumAddress === INTERNAL_ACCOUNT_ETH_ADDR
+                ? getPartiallyHiddenHermezAddress(txData.toBjj || txData.toBJJ)
```

Comment:
Both `BJJ` and `Bjj` exist? If so isn't this an API issue?

Reply:
Yup, going to create an issue there, but we need this patch in the meantime